code/upaya/app/views/shared/_upaya_legal_notice.html.slim

```diff
@@ -0,0 +1,15 @@
+.panel.panel-default
+  .panel-body.legal-notice
+    ul.horizontal-menu
+      li
+        = link_to 'Privacy Act Statement', 'terms#privacy', data: { toggle: "modal", target: "#privacyModal" }, class: 'modalpop'
+      li
+        = link_to 'Paperwork Reduction Act Reporting Burden', 'terms#pra',  data: { toggle: "modal", target: "#praModal" }, class: 'modalpop'
+      li
+        = link_to 'Accessibility Policy', "#{user_signed_in? && user.role != 'user' ?  'http://upaya.18f.gov/pages/accessibility.aspx' : 'http://upaya.18f.gov/accessibility'}",  target: '_blank'
+    h4 Department of Homeland Security Consent
+    = render 'shared/dhs_text'
```

Comment:
nope, really we just want some placeholder stuff until we have a better idea of what the content will look like for the PoC CC @brendansudol @amoose